src/utils/next.ts

```diff
@@ -26,7 +26,7 @@ export type ScaffoldedProps = RegularProps & {
 
 export type ScaffoldedContext = GetServerSidePropsContext & {
     apiFetch: (path : string, init? : RequestInit) => Promise<Response>;
-    user: ZetkinZResult | null;
+    user: unknown;
```

Comment:
This should be typed as `ZetkinUser`.

Reply:
Yes I tried that, but get the following:

`Type 'unknown' is not assignable to type 'ZetkinUser'.`